### ISSUE TITLE
plot_nodal_solution doesn't create the animation when parameter off_screen=True

### ISSUE THREAD

**Issue by Rfaelv** [pyansys/PyMAPDL-reader#30]
I try:

result.animate_nodal_solution(6, movie_filename='movie6.gif', cpos=cpos, loop=False,  off_screen=True, background='w')

and I got a static image of the first frame.

**Comment by akaszynski** [comment]
Could you please run the following?
```
>>> from ansys.mapdl.core import Report
>>> print(Report())
```

And let me know what the output of the report is?

**Comment by Rfaelv** [comment]
![image](https://user-images.githubusercontent.com/82391210/120071222-a4dad700-c064-11eb-8192-e7a5ea9768e5.png)

**Comment by akaszynski** [comment]
Found the bug.  Expect this fixed in the next release 0.51.1.  Sorry it took so long to get around to.

**Comment by Rfaelv** [comment]
Thanks!
